tg_bot/modules/sql/welcome_sql.py

```diff
@@ -17,11 +17,13 @@ class Welcome(BASE):
 
     custom_welcome = Column(UnicodeText, default=DEFAULT_WELCOME)
     welcome_type = Column(Integer, default=Types.TEXT.value)
-
+    
     custom_leave = Column(UnicodeText, default=DEFAULT_GOODBYE)
     leave_type = Column(Integer, default=Types.TEXT.value)
 
     clean_welcome = Column(BigInteger)
+    welcome_mute = Column(Boolean, default=False)
```

Comment:
You've added a column, what's the SQL for it? It doesn't get added automatically.